charts/osm/templates/mutatingwebhook.yaml

```diff
@@ -3,17 +3,28 @@ kind: MutatingWebhookConfiguration
 metadata:
   labels:
     app: osm-controller
-  name: osm-webhook-{{.Values.OpenServiceMesh.meshName}}
+  name: arc-osm-webhook-{{.Values.OpenServiceMesh.meshName}}
```

Comment:
You probably need to update the e2e for this https://github.com/Azure/osm-azure/blob/main/test/bats/test.bats#L19

Reply:
Since the deployment does not create any namespace yet, pending this issue: https://github.com/openservicemesh/osm/issues/1817
we need to update https://github.com/Azure/osm-azure/blob/main/Makefile#L23 with 
`kubectl create ns arc-osm-system` then 
`helm install --namespace arc-osm-system` 
you also need to update the e2e script to look for resources in the `arc-osm-system` namespace now.

cc @SanyaKochhar FYI

Reply:
Should we create an issue for the part to update the makefile ? I